ui/app/components/app/asset-list-item/asset-list-item.js

```diff
@@ -40,6 +58,36 @@ const AssetListItem = ({
     )
     : null
 
+  const sendTokenButton = useMemo(() => {
+    if (tokenSymbol === 'ETH') {
```

Comment:
Technically there is nothing stopping a token from using `ETH` as its symbol. Certainly a user can input one.

Instead the existence of the `tokenAddress` prop would probably be the best way to determine whether this asset list item is for a token or not.